<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntryDialog</class>
 <widget class="QDialog" name="EntryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lRecipeName">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lERecipeName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lDesc">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPlainTextEdit" name="pTextEditDescription"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lIngredients">
       <property name="text">
        <string>Ingredients</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTableView" name="tVIngredients"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddNew">
       <property name="text">
        <string>Add new recipe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddIngredient">
       <property name="text">
        <string>Add ingredient</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemIngredient">
       <property name="text">
        <string>Remove ingredient</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
